### ISSUE TITLE
Helm 3: errors regarding Chart.yaml metadata are not clear

### ISSUE THREAD

**Issue by karuppiah7890** [helm/helm#5653]
Output of `helm version`:

```
version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"1b77848c89e8df414aea055cca5d1a1997a62f0c", GitTreeState:"clean"}
```

Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-26T00:05:06Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.8-gke.6", GitCommit:"394ee507d00f15a63cef577a14026096c310698e", GitTreeState:"clean", BuildDate:"2019-03-30T19:31:43Z", GoVersion:"go1.10.8b4", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): GKE

I tried to do `helm template` and this is what I got:

```
$ helm template redis                                  
Error: metadata apiVersion is required
```

On checking I noticed that the error was regarding my `Chart.yaml` file not having `apiVersion` field. But the error was not clear about the origin of the issue, due to which I started thinking if my template yamls missed the `apiVersion` field which is part of the k8s resource yamls. I think the error message could tell about the origin, for example

```
$ helm template redis                                  
Error: Chart.yaml: metadata apiVersion is required
```

**Comment by hickeyma** [comment]
@karuppiah7890 I think this is a good point. The implication is that it is the Chart metadata and not the k8s resource metadata. It is however not very clear.

This is a good first issue if you would like to give it a go. The changes are in: https://github.com/helm/helm/blob/dev-v3/pkg/chart/metadata.go#L71

I think the error message might read better as:
`Error: Chart 'apiVersion' metadata is required in Chart.yaml. ` The other message should also be updated.

**Comment by cpanato** [comment]
@hickeyma can i try this one?

**Comment by hickeyma** [comment]
@cpanato I think we should let @karuppiah7890 reply first. I had already asked in https://github.com/helm/helm/issues/5653#issuecomment-487525914.

**Comment by cpanato** [comment]
@hickeyma sure thing.

**Comment by karuppiah7890** [comment]
Sure! I would love to contribute! :) I will raise a PR and ping here! :)

**Comment by karuppiah7890** [comment]
Create a PR #5660 for the issue! 😁

**Comment by hickeyma** [comment]
Closing as implemented in #5560.
